hugr-core/tests/snapshots/model__roundtrip_const.snap

```diff
@@ -19,40 +29,41 @@ expression: "roundtrip(include_str!(\"../../hugr-model/tests/fixtures/model-cons
 (define-func example.make-pair
   []
   [(adt
-     [[(@ arithmetic.float.types.float64) (@ arithmetic.float.types.float64)]])]
+     [[(@ collections.array.array 5 (@ arithmetic.int.types.int 6))
+       (@ arithmetic.float.types.float64)]])]
   (ext)
   (dfg
     [] [%0]
     (signature
       (->
         []
         [(adt
-           [[(@ arithmetic.float.types.float64)
+           [[(@ collections.array.array 5 (@ arithmetic.int.types.int 6))
              (@ arithmetic.float.types.float64)]])]
         (ext)))
     (const
       (tag
         0
         [(@
-           compat.const-json
-           (@ arithmetic.float.types.float64)
-           "{\"c\":\"ConstF64\",\"v\":{\"value\":2.0}}"
-           (ext arithmetic.float.types))
-         (@
-           compat.const-json
-           (@ arithmetic.float.types.float64)
-           "{\"c\":\"ConstF64\",\"v\":{\"value\":3.0}}"
-           (ext arithmetic.float.types))])
+           collections.array.const
+           5
+           (@ arithmetic.int.types.int 6)
+           [(@ arithmetic.int.const 6 1)
+            (@ arithmetic.int.const 6 2)
+            (@ arithmetic.int.const 6 3)
+            (@ arithmetic.int.const 6 4)
+            (@ arithmetic.int.const 6 5)])
+         (@ arithmetic.float.const-f64 -3.0)])
       [] [%0]
       (signature
         (->
           []
           [(adt
-             [[(@ arithmetic.float.types.float64)
+             [[(@ collections.array.array 5 (@ arithmetic.int.types.int 6))
                (@ arithmetic.float.types.float64)]])]
           (ext))))))
 
-(define-func example.f64
+(define-func example.f64-json
```

Comment:
is this still using json?

Reply:
Yes: if you check `hugr-model/tests/fixtures/model-const.edn` you see that it loads two json constants. Only the unknown one is exported again as json. This is to have the json fallback path in the test coverage.